src/ffjord.jl

```diff
@@ -0,0 +1,56 @@
+abstract type CNFLayer <: Function end
+Flux.trainable(m::CNFLayer) = (m.p,)
+
+struct FFJORD{M,P,RE,ContinuousUnivariateDistribution,Bool,T,A,K} <: CNFLayer
+    model::M
+    p::P
+    re::RE
+    baseDist::ContinuousUnivariateDistribution
+    MonteCarlo::Bool
+    tspan::T
+    args::A
+    kwargs::K
+
+    function FFJORD(model,tspan,args...;p = nothing,baseDist=nothing,MonteCarlo=nothing,kwargs...)
+        _p,re = Flux.destructure(model)
+        if p === nothing
+            p = _p
+        end
+        if baseDist === nothing
+            baseDist = Normal(0,1)
+        end
+        if MonteCarlo === nothing
+            MonteCarlo = false
+        end
```

Comment:
```suggestion
```

There's no need to treat these in a separate manner

Reply:
What I had in mind is that the user could choose, for example, to use the default base distribution and set monte_carlo=true , to use an alternative distribution and the default monte_carlo, etc. 

monte_carlo and basest would be independent in this sense.

Reply:
no, normal keyword argument handling works just fine in that case. You don't need `=== nothing` and then special handling.

Reply:
see that https://github.com/SciML/DiffEqFlux.jl/pull/287#discussion_r437083643 sets those values as the default. `p` is handled differently because `destructure` is where `p` comes from, unless the user passed in `p`: that situation is complex enough to not just be a keyword argument.

Reply:
oh, didn't see that it was changed to false